### ISSUE TITLE
[FEAT] Support PDF3 accessibility tags for reading order (W3C)

### ISSUE THREAD

**Issue by Benvii** [datalab-to/marker#792]
## ✨ Is your feature request related to a problem?

Some companies makes PDFs with accessibility standard, on of them is tags to specify the document reading order (which is one of the complex part when parsing PDFs).

Standard is described here : https://www.w3.org/TR/WCAG20-TECHS/PDF3.html 

## 💡 Describe the Solution You'd Like

Marker (and pdfium), could take advantage of those tags to determine the reading order this would strongly help / fixe wrong or partially correct layout detection.

## 📋 Alternatives Considered

NA

## 🧩 Additional Context

I'm not sure if it should be a marker feature and / or a pdfium feature.
